src/SequenceManager.cc

```diff
@@ -327,16 +343,34 @@ void SequenceManager::rename_sequence(UniqueId unique_id, const SequenceName& ne
         throw Error(gul14::cat("Cannot rename folder ", old_path.string(),
             " to ", new_path.string(), ": ", error.message()));
     }
+
+    // commit to local repository
+    auto commit_msg = stage_files_in_directory(old_seq_on_disk.path, "");
+    commit_msg += stage_files_in_directory(make_sequence_filename(new_name, unique_id), "");
+    if (commit_msg != "")
+        git_repo_.commit(gul14::cat("Rename ", old_seq_on_disk.name.string(), " to ", new_name.string(), ":", commit_msg));
```

Comment:
I do not understand this entirely: Doesn't `stage_files_in_directory()` stage everything *recursively*? Then why do we need the second call to it?

Reply:
The function stages everything in the directory which is defined in the first parameter (eg. old_seq_on_disk.path)
As we rename the sequence, we are dealing with 2 directories: old sequence name and new_sequence name.

I do not want to stage everything, only the current sequence the user is working on. In case multiple instances of taskomat GUI are used and 2 users parallel working on different sequences. And I want distinctive commits for every sequence